girder/api/docs.py

```diff
@@ -112,13 +112,25 @@ def removeRouteDocs(resource, route, method, info, handler):
                 del routes[resource]
 
 
-def addModel(name, model):
+def addModel(resource, name, model):
     """
-    This is called to add a model to the swagger documentation.
+    Add a model to the Swagger documentation.
 
+    :param resource: The type of existing resource to add the model to.
+    :param resource: str
     :param name: The name of the model.
     :type name: str
     :param model: The model to add.
     :type model: dict
```

Comment:
So the way I see this, Brian is more correct in how we _should_ use Swagger for this. I think for now, resource should be the last argument and default to a value that will cause the body of this function to set it for all current and future resources on the models dict.

Does that make sense?